botorch/optim/initializers.py

```diff
@@ -363,7 +373,7 @@ def gen_batch_initial_conditions(
                         X_rnd_nlzd = torch.rand(
                             n, q, bounds_cpu.shape[-1], dtype=bounds.dtype
                         )
-                    X_rnd = bounds_cpu[0] + (bounds_cpu[1] - bounds_cpu[0]) * X_rnd_nlzd
+                    X_rnd = X_rnd_nlzd * (bounds_cpu[1] - bounds_cpu[0]) + bounds_cpu[0]
```

Comment:
you could use `unnormalize` from https://github.com/pytorch/botorch/blob/2e143c9c7876f4d3534cc46e56a80bd8120f8d62/botorch/utils/transforms.py#L96 here